### PR TITLE
Play inspect viewer beginning to work

### PR DESCRIPTION
**Pull request by corruptmemory**
The play inspect tab is beginning to work.  When inspecting an individual request the details seem hard-coded.  We can either merge this and I can work on getting request details working or we can add commits to this PR to get details working.

### CODE REVIEW

**Comment by havocp** [comment]
if this is an improvement I'm all for merging it but I think it could use a @henrikengstrom or @warry review

File: ui/app/console/ClientModuleHandler.scala [hunk 1]
```diff
@@ -26,16 +26,20 @@ object ClientModuleHandler {
...
-    case "actors" => Some(ActorsModule)
-    case "actor" => Some(ActorModule)
-    case "requests" => Some(RequestsModule)
-    case "request" => Some(RequestModule)
-    case "deviations" => Some(DeviationsModule)
-    case "deviation" => Some(DeviationModule)
-    case "lifecycle" => Some(LifecycleModule)
-    case _ => None
+  def fromString(in: String): Option[Handler] = {
+    println(s"---- fromString($in)")
```
Comment (henrikengstrom) [inline ui/app/console/ClientModuleHandler.scala]: Might want to remove this?

File: ui/app/console/handler/PlayRequestHandler.scala [hunk 2]
```diff
@@ -49,7 +49,7 @@ trait PlayRequestHandlerBase extends RequestHandler[PlayRequestHandler.PlayReque
         } yield {
           usePlayRequestStats(sender, id, playStats, actorInfo)
         }
-      case None => log.warning("Cannot call Play request detail information without proper trade id.")
+      case None => log.warning("Cannot call Play request detail information without proper trace id.")
```
Comment (henrikengstrom) [inline ui/app/console/handler/PlayRequestHandler.scala]: why u no want "trade id"?!?!

**Comment by henrikengstrom** [comment]
LGTM apart from minor comment about ``println``

**Comment by henrikengstrom** [comment]
Fixed the ``println`` issue. Merging now.

File: ui/app/console/ClientModuleHandler.scala (hunk 1 above)
Comment (corruptmemory) [inline ui/app/console/ClientModuleHandler.scala]: Um, oops.  Will clean up in a new PR

**Pull request merged**